### ISSUE TITLE
Non-text Slider elements do not meet minimum contrast requirements

### ISSUE THREAD

**Issue by lloydstubber** [cultureamp/kaizen-design-system#1917]
TLDR; Speak with a designer and get the elements to the right level of contrast

**Type**: WCAG 2
**Principle**: Perceivable
**Guideline:1.4** Distinguishable
**Success Criteria**: 1.4.11 Non-text Contrast (AA)
**Level**: AA
**Platform**: Web
**Severity**: Medium
**URL**: https://intopia.cultureamp.com/performance/1-1/6ce44465-ac60-426c-82f8-94d5d8c3ca92

### What's the issue?
Non-text objects do not meet the minimum contrast requirements.
<img width="645" alt="Screen Shot 2021-09-02 at 9 55 36 am" src="https://user-images.githubusercontent.com/18339250/131760362-66dc7984-3ce7-4c06-bd2b-0b091e189ec0.png">

Element | Foreground | Background | Minimum | Actual
-- | -- | -- | -- | --
Slider background | #E6E5E5 | #FFFFFF | 3:1 | 1.3:1
Slider “spokes” | #E6E5E5 | #FFFFFF | 3:1 | 1.3:1

### Why it matters
Good contrast is useful for all users but is especially important for people with low vision or colour blindness and people viewing your website on a mobile device in a bright or sunny location.

### Our recommendation
Make sure that any visual information necessary to identify an active user interface component, understand how it works or determine its state has a contrast ratio of at least 3:1 against the adjacent background. Suggested colour for slider grey: #828181

Tip: The Tanaguru Contrast Finder (http://contrast-finder.tanaguru.com/ ) can identify the closest shade to the original colour that meets the minimum contrast requirements.
